### PR TITLE
Release note Oct

### PR DESCRIPTION
**Pull request by v-himkam**

### CODE REVIEW

File: Release notes/LatestReleaseNotes.md [hunk 1]
```diff
@@ -1,50 +1,47 @@
...
 
 ### Security controls updates
 * New controls:
-   The following new control have been added in this release:
-   * ```ADO_Repository_AuthZ_Dont_Grant_BuildSvc_Permission_On_Branch```: Do not grant build service groups excessive permissions on repository branches
-
+    The following new control have been added in this release:
+	* ``ADO_Environment_AuthZ_Enable_PreDeployment_Approval``: Environments for production deployments must haveapprovals enabled.
+	* ``ADO_Environment_AuthZ_Review_PreDeployment_Approvers``: Approvers on environment must be periodically reviewed.
+	* ``ADO_Environment_SI_Use_Good_Branch_Hygiene``: All deployments to production environments must be done fromstandard branches.
```
Comment (SaikumarRagala) [inline Release notes/LatestReleaseNotes.md:25]: from standard

File: Release notes/LatestReleaseNotes.md [hunk 2]
```diff
@@ -1,50 +1,47 @@
...
+	* ``ADO_Project_AuthZ_Limit_Build_Admin_Count``: Ensure that there are not many build administrators in yourproject.
```
Comment (SaikumarRagala) [inline Release notes/LatestReleaseNotes.md:26]: your project

File: Release notes/LatestReleaseNotes.md [hunk 3]
```diff
@@ -1,50 +1,47 @@
...
+	* ``ADO_Organization_AuthZ_Disable_Request_Access``: Disable request access policy in your organization.
+	* ``ADO_Repository_DP_Enable_Credentials_And_Secrets_Policy``:  Enable 'Check for credentials and other secrets' atrepo policy.
```
[the remark this answers is not in the document]
Comment (v-himkam) [reply Release notes/LatestReleaseNotes.md]: done

File: Release notes/LatestReleaseNotes.md [hunk 4]
```diff
@@ -1,50 +1,47 @@
...
 
 * Changes to existing controls :
-   * ```ADO_Feed_AuthZ_Dont_Grant_BuildSvcAcct_Permission ```
-   * ```ADO_VariableGroup_DP_No_PlainText_Secrets_In_Variables```
-   * ```ADO_AgentPool_DP_No_Secrets_In_Capabilities```
-   * ```ADO_Repository_AuthZ_Dont_Grant_BuildSvcAcct_Permissio```
-   * ```ADO_ServiceConnection_SI_Dont_Share_Across_Projects ```
-   * ```ADO_AgentPool_AuthZ_Dont_Enable_Auto_Provisioning ```
+	* ``ADO_Project_AuthN_Use_ALT_Accounts_For_Admin``: Can now be configured to check for non SC-ALT admins in the following groups as well:Build admins, Release admins and Endpoint admins
+	* ``ADO_Project_AuthZ_Revoke_Admin_Access_for_Inactive_Users``: Can now be configured to check for inactive admins in the followinggroups as well: Build admins and Release admins
```
[the remark this answers is not in the document]
Comment (v-himkam) [reply Release notes/LatestReleaseNotes.md]: done

**Pull request merged**